charts/alfresco-sync-service/templates/deployment.yaml

```diff
@@ -0,0 +1,139 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "alfresco-sync-service.fullname" . }}
+  labels:
+    {{- include "alfresco-sync-service.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "alfresco-sync-service.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+{{ toYaml .Values.global.strategy.rollingUpdate | indent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "alfresco-sync-service.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+        {{- with .Values.podAnnotations }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- include "alfresco-common.component-pod-security-context" .Values | indent 4 }}
+      serviceAccountName: {{ include "alfresco-sync-service.serviceAccountName" . }}
+      {{- include "alfresco-common.imagePullSecrets" . | indent 6 }}
+      terminationGracePeriodSeconds: 60
```

Comment:
docker image is not handling correctly shutdown requests?

worth having a value to configure it imho

Reply:
I'm not sure tbh. I've just kept what was there before. But after taking a quick look it seems the docker image is using a wrapper script that does some substitutions before spawning the server.... So I can only imagine that this is indeed not handled properly by docker